addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1016,6 +1016,25 @@ options.registry.OptionsTab = options.Class.extend({
             hsl.lightness);
         return ColorpickerWidget.convertRgbaToCSSColor(adjustedGrayRGB.red, adjustedGrayRGB.green, adjustedGrayRGB.blue);
     },
+    /**
+     * @override
+     */
+    async _renderCustomXML(uiFragment) {
+        await this._super(...arguments);
+        const rangeExtraSaturation = uiFragment.querySelector('we-range[data-param="gray-extra-saturation"]');
+        if (rangeExtraSaturation) {
+            const defaultSaturationOfGrey = _.range(100, 1000, 100).map(id => {
+                const gray = weUtils.getCSSVariableValue(`base-${id}`);
+                const grayRGB = ColorpickerWidget.convertCSSColorToRgba(gray);
+                const hsl = ColorpickerWidget.convertRgbToHsl(grayRGB.red, grayRGB.green, grayRGB.blue);
+                return hsl.saturation;
+            });
+            const maxSaturation = _.max(defaultSaturationOfGrey);
+            const minSaturation = _.min(defaultSaturationOfGrey);
+            rangeExtraSaturation.dataset.max = 100 - maxSaturation;
+            rangeExtraSaturation.dataset.min = -minSaturation;
```

Comment:
I don't think those are the right values. Should not it be `100 - minSaturation` and `-maxSaturation`? :thinking:

Reply:
Sorry, my bad 😶